doc/source/user/feature-classification/feature-matrix-lb.ini

```diff
@@ -0,0 +1,124 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+[driver.elb]
```

Comment:
missing parameters:
tenant_id, provider

Reply:
actually even provider is not really necessary. Here we list what of the Octavia API is supported by individual drivers. "Provider" attribute as such is only attribute of the Octavia API used to identify which backend is going to be used. It is of no relevance for the drivers themselves.